app/controllers/core_data_connector/relationships_controller.rb

```diff
@@ -25,17 +25,21 @@ def base_query
       # Relationships should always to scoped to a project model relationship. For inverse relationships, we'll
       # use the related record.
       if params[:inverse]
-        query = Relationship.where(
-          project_model_relationship_id: params[:project_model_relationship_id],
-          related_record_id: params[:record_id],
-          related_record_type: params[:record_type]
-        )
+        query = Relationship
```

Comment:
Related records are currently display on the front-end in a list view that allows column sorting.

![Screenshot 2025-04-08 at 2 38 05 PM](https://github.com/user-attachments/assets/1822f740-590f-4dc5-9353-f14133957561)

Will forcing a sort by `order` here prevent column sorting from working?

Reply:
Looks like it does. https://github.com/performant-software/core-data-cloud/issues/425 asked for the authenticated controller to sort by `order` by default but I don't see a way to do that without affecting the ability to sort by other attributes in the UI.

Would it make more sense to update the Core Data UI to show the order values in a tab, and maybe have the table sorted that way by default as opposed to hardcoding the sorting in the controller?